grails-app/views/job/show.gsp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="${resource(dir: 'css', file: 'imos.css')}" type="text/css">
+    </head>
+    <body>
+        <div class="imosHeader">
+            <div class="container">
+                <a class="btn" role="button" href="https://imos.aodn.org.au/imos123/home?uuid=8af21108-c535-43bf-8dab-c1f45a26088c">
+                    <img src="http://static.emii.org.au/images/logo/IMOS-Ocean-Portal-logo.png" alt="IMOS logo">
+                </a>
+            </div>
+        </div>
+        <div class="container">
+            <dl>
+                <dt><g:message code="job.id.label" default="Id"/></dt>
+                <dd>${job.uuid}</dd>
```

Comment:
Following on from above, might be good to introduce a new tag, say "colonSeparatedKeyValue", which takes the label and value, and turns it into `<dt>`, `<dd>`?  It could also potentially take care of the `<g:if>` clause...

Sorry I hope I'm not being pedantic, but I know that there is **a lot** of duplication in our GSPs for other projects, and it'd be good to not make the same mistakes here given that we've got a clean slate!